DXRMapFixups/DeusEx/Classes/DXRFixupM02.uc

```diff
@@ -159,6 +162,32 @@ function PreFirstEntryMapFixes()
             Spawn(class'PlaceholderItem',,, vectm(15,-2972,123)); //Kitchen counter
             Spawn(class'PlaceholderItem',,, vectm(-853,-3148,75)); //Crack next to Paul's bed
         }
+
+        oldSeed = SetGlobalSeed("'Ton Hotel Junkie Names " $ dxr.seed);
+        foreach AllActors(class'#var(DeusExPrefix)Carcass', dead) {
+            if (dead.Name == 'JunkieFemaleCarcass0') {
+                if (class'DXRMenuScreenNewGame'.static.HasLDDPInstalled()) {
+                    dead.itemName = "Dead Body (Jenna)";
+                } else {
+                    dead.itemName = "Dead Body (" $ class'DXRNames'.static.RandomName(dxr) $ ")";
+                }
+            } else if (dead.Name == 'JunkieMaleCarcass0') {
+                if (class'DXRMenuScreenNewGame'.static.HasLDDPInstalled()) {
+                    dead.itemName = "Dead Body (Kyle)";
+                } else {
+                    dead.itemName = "Dead Body (" $ class'DXRNames'.static.RandomName(dxr) $ ")";
+                }
+            }
+        }
+        foreach AllActors(class'HumanCivilian', hc) {
+            if (hc.Name == 'LDDPHotelAddict0') {
+                hc.FamiliarName = class'DXRNames'.static.RandomName(dxr);
```

Comment:
I'm not sure I understand why we're explicitly giving this npc a random name and then marking them as important?